### ISSUE TITLE
Automate releases with semantic-release

### ISSUE THREAD

**Issue by thibaudcolas** [thibaudcolas/draftjs-filters#6]
...when it is ready. Config:

```json
{
"release": {
    "branch": "master",
    "verifyConditions": [
      "@semantic-release/changelog",
      "@semantic-release/npm",
      "@semantic-release/git",
      "@semantic-release/github"
    ],
    "getLastRelease": "@semantic-release/npm",
    "analyzeCommits": {
      "preset": "angular",
      "releaseRules": [
        {
          "type": "docs",
          "scope": "README",
          "release": "patch"
        },
        {
          "type": "refactor",
          "release": "patch"
        }
      ]
    },
    "verifyRelease": [],
    "generateNotes": {
      "preset": "angular"
    },
    "publish": [
      "@semantic-release/changelog",
      "@semantic-release/npm",
      {
        "path": "@semantic-release/git",
        "assets": [
          "package.json",
          "package-lock.json",
          "CHANGELOG.md"
        ]
      },
      {
        "path": "@semantic-release/github",
        "assets": [
          "package.json",
          {
            "path": "dist/draftjs-filters.cjs.js",
            "label": "CommonJS"
          },
          {
            "path": "dist/draftjs-filters.esm.js",
            "label": "ES modules"
          }
        ]
      }
    ]
  },
  "publishConfig": {
    "tag": "next"
  }
}
```

> Releases are automated with [semantic-release](https://github.com/semantic-release/semantic-release) on every push to `master`, based on the content of the commits. New versions are pushed to the `next` tag on npm, and must then manually be promoted to `latest`.

**Comment by thibaudcolas** [comment]
For future reference – what didn't work was adding the changelog generation, then committing the changelog to git.

This was in v12 of semantic-release.

**Comment by thibaudcolas** [comment]
:tada: This issue is fixed in v2.0.0, available on npm: [draftjs-filters@2.0.0](https://www.npmjs.com/package/draftjs-filters).

<p align="right">
Generated by :package::rocket: <a href="https://github.com/semantic-release/semantic-release">semantic-release</a>
</p>
